examples/layers/preprocessing/classification/random_height_demo.py

```diff
@@ -0,0 +1,21 @@
+"""
+random_height_demo.py shows how to use the RandomHeight preprocessing layer for
+object detection.
+"""
+import demo_utils
```

Comment:
Sharing the same set of comments with the other RandomWidth PR

Reply:
Right, okay. Have removed it!